.env.development

```diff
@@ -18,3 +18,4 @@ SITE_NAME='edX'
 APP_ID=''
 MFE_CONFIG_API_URL=''
 ADDITIONAL_METADATA_REQUIRED_FIELDS='{}'
+IS_NEW_SLUG_FORMAT_ENABLED='false'
```

Comment:
nope, we can use simple `false` as well. But on accessing it with process.env.IS_NEW_SLUG_FORMAT_ENABLED it will always converted to string whether it is in quotes or not. So, i decided to put the quotes here.